runtime/near-evm-runner/src/utils.rs

```diff
@@ -183,66 +187,497 @@ pub fn format_log(topics: Vec<H256>, data: &[u8]) -> std::result::Result<Vec<u8>
     Ok(result)
 }
 
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub enum Type {
```

Comment:
What is this type and why is it public?
 - Add a comment
 - Rename to something meaningful or wrap into a module, so it has a context.
 - Consider making it non public visibility.

Reply:
I think it's better to name to ArgType or EvmArgType, make private. it is "Type" and public because it's copied from eip-712 crate :D

Reply:
It need to be pub to construct and compare in test. pub(crate) maybe better, but we can simply make the whole utils mod private to near-evm-runner